### ISSUE TITLE
A11y: Return keyboard focus to category after closing submenu

### ISSUE THREAD

**Issue by ravenokko** [shopware/platform#8842]
### Shopware Version

6.7

### Affected area / extension

Platform(Default)

### Actual behaviour

If you close the subcategory menu with a keyboard, the tabbing order remains correct, but the focus indicator is temporarily missing.

### Expected behaviour

When you close the subcategory menu with a keyboard, the focus indicator should return to the parent category. This should occur for all category types (Page, Link, Structuring element).

### How to reproduce

- Create a category of any type with a subcategory of any type. 
- Using only the keyboard, navigate to that category in the storefront with a keyboard and open the subcategory menu
- Close the subcategory menu
